FAQS:[CDS-97308]: Addition of CD Faqs (#6896)
harness/developer-hub: docs/continuous-delivery/deploy-srv-diff-platforms/aws/aws-deployment-faqs.md

'''

### Does Harness support viewing the rendered launch template for ASG deployments after all the Harness expressions have been evaluated?

> No, Harness does not store the rendered launch template as an output variable, nor does it display the rendered launch templates in the console logs. We only indicate that the ASG configuration is created. Please refer more on this in [Harness ASG services](https://developer.harness.io/docs/continuous-delivery/deploy-srv-diff-platforms/aws/asg-tutorial/#harness-asg-services)

Reviewer: ```suggestion
No, Harness does not store the rendered launch template as an output variable, nor does it display the rendered launch templates in the console logs. We only indicate that the ASG configuration is created. For more details, go to [Harness ASG services](https://developer.harness.io/docs/continuous-delivery/deploy-srv-diff-platforms/aws/asg-tutorial/#harness-asg-services).
```
